### PR TITLE
Make minor improvements to `build-passed.svg`

### PR DESCRIPTION
**Pull request by alrra**
- Group the `<text>` elements together and only apply the font related styles on that group.
- Make other minor improvements (e.g.: use `0` instead of `0%`).

### CODE REVIEW

File: build-passed.svg [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
-  <rect rx='4px' width='77px' height='18px' style='fill:url(#ggrey)'/>
-  <rect rx='4px' x='32px' width='45px' height='18px' style='fill:url(#ggreen)'/>
-  <rect x='32px' width='5px' height='18px' style='fill:url(#ggreen)'/>
-  <text x='5px' y='12.5px' style='stroke:rgba(0,0,0,0.3)'>build</text>
-  <text x='5px' y='12px'>build</text>
-  <text x='39px' y='12.5px' style='stroke:rgba(0,0,0,0.3)'>passed</text>
-  <text x='39px' y='12px'>passed</text>
+  <rect rx="4" width="77" height="18" fill="url(#a)"/>
+  <rect rx="4" x="32" width="45" height="18" fill="url(#b)"/>
+  <rect x="32" width="5" height="18" fill="url(#b)"/>
```
Comment (mathiasbynens) [inline build-passed.svg]: Ah seems you just forgot to replace them here?

Comment (alrra) [inline build-passed.svg]: yes, I've rushed a little bit, should be fixed now. :)

**Comment by alrra** [comment]
@espadrine @mathiasbynens [re-added](https://github.com/espadrine/gh-badges/commit/275680d01d0ad804892e4f5338e343441a9aaf96#commitcomment-4968838) the `width="77px" height="18px"` part.

**Comment by mathiasbynens** [comment]
@espadrine Did you do anything special to merge this, or does GitHub’s Merge button functionality finally not introduce those annoying superfluous merge commits anymore?

**Comment by espadrine** [comment]
On Thu, Jan 2, 2014 at 3:34 PM, Mathias Bynens notifications@github.com wrote:

> @espadrine Did you do anything special to merge this, or does GitHub’s Merge button functionality finally not introduce those annoying superfluous merge commits anymore?

Heh. I rebased it. Github still introduces a merge commit.

**Comment by mathiasbynens** [comment]
Too bad. Got me excited there for a sec :)

**Pull request merged**